tests/e2e/scenarios/ha/templates/kill-router-pods.sh.j2

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+if [ -f "/tmp/stop-killing-router-pod" ]; then
+    rm "/tmp/stop-killing-router-pod"
+fi
+
+JOBSTATUS=$(oc --kubeconfig {{ kubeconfig_1 }} -n {{ namespace_prefix }}-west-ha get job locust-job  -o custom-columns=COMPLETED:.status.succeeded,ERROR:.status.failed --no-headers | grep -c "1")
```

Comment:
I believe that you should use `kubectl` instead as the CI probably does not have `oc`.

Reply:
changed to Kubectl